apps/storybook/stories/react-essentials/useUpdateLoop.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Column,
+  IconButton,
+  PauseIcon,
+  PlayArrowIcon,
+  Row,
+  ShapedBox,
+  StopIcon,
+  background,
+  shape,
+  size,
+} from "@codedazur/react-components";
+import { Frame, useUpdateLoop } from "@codedazur/react-essentials";
+import { Meta, StoryObj } from "@storybook/react";
+import { useEffect, useMemo, useRef, useState } from "react";
+import { DebugOverlay } from "../../components/DebugOverlay";
+import docs from "./useUpdateLoop.docs.mdx";
+import styled from "styled-components";
+
+interface UseUpateLoopArgs {
+  timeScale?: number;
+  targetFps?: number;
+}
+
+const meta: Meta<UseUpateLoopArgs> = {
+  title: "react-essentials/useUpdateLoop",
+  parameters: {
```

Comment:
We don't need this syntax anymore afaik. The docs will appear by itself as a new tab. Let's double check this.